singer/src/main/java/com/pinterest/singer/common/SingerSettings.java

```diff
@@ -94,6 +95,10 @@
   //environment is production by default
   private static Environment environment = new Environment();
 
+  // loggingAuditClient is used to send LoggingAuditEvent if LoggingAudit feature is enabled and
+  // a TopicAuditConfig is set for a given logStream.
+  public static LoggingAuditClient loggingAuditClient = null;
```

Comment:
why is this variable public? please use getters and setters

Reply:
change to getter and setter